src/pubsub.ts

```diff
@@ -70,6 +70,7 @@ export type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
 
 export interface ClientConfig extends gax.GrpcClientOptions {
   apiEndpoint?: string;
+  useFakeCredentials?: boolean;
```

Comment:
I added a comment on the bug, but we should seriously consider alternatives to just adding another property like this. If we do end up adding it, let's not call it "useFakeCredentials."